### PR TITLE
Adding clicommandlineparser and cli-migrate to dnceng mirroring

### PR DESCRIPTION
**Pull request by livarcocc**
@chcosta

### CODE REVIEW

**Comment by chcosta** [comment]
@dagood @mmitche , can you review / merge?

**Comment by livarcocc** [comment]
Once this is merged, can we trigger a manual mirroring, so that the repo gets some content ahead of the next PR being merged?

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -854,6 +854,8 @@
         "https://github.com/dotnet/sdk/blob/rel/**/*",
         "https://github.com/dotnet/sdk/blob/release/**/*",
         "https://github.com/dotnet/dotnet-cli-archiver/blob/master/**/*",
+        "https://github.com/dotnet/cli-migrate/blob/master/**/*",
+        "https://github.com/dotnet/clicommandlineparser/blob/master/**/*",
```
Comment (dagood) [inline Maestro/subscriptions.json]: Why no release branches?

**Comment by dagood** [comment]
Maestro doesn't have a mechanism for that (https://github.com/dotnet/versions/issues/99), but you can queue the underlying build yourself and fill in the parameters manually. I think https://github.com/dotnet/versions/tree/master/Maestro#subscriptionsjson is still an accurate enough description of the file.

If possible, it would be much easier to push a dummy commit. (Unfortunately it does have to make an actual change, because the `**/*` has to match something. https://github.com/dotnet/versions/issues/110)

File: Maestro/subscriptions.json (hunk 1 above)
Comment (dagood) [inline Maestro/subscriptions.json]: Do you know if these have Maestro webhooks set up for them? (A repo admin needs to set this up before first use.)

Instructions: https://github.com/dotnet/core-eng/blob/master/Documentation/Maestro/web-hooks.md

Comment (livarcocc) [reply Maestro/subscriptions.json]: We don’t have a release branch moving to VSTS right now.

Comment (livarcocc) [reply Maestro/subscriptions.json]: Yes. I added those already.

Comment (dagood) [reply Maestro/subscriptions.json]: Ok. It'd be fine to add the trigger preemptively, if you want, but this is fine.

**Pull request merged**